ngraph/test/type_prop/proposal.cpp

```diff
@@ -22,16 +22,19 @@
 using namespace std;
 using namespace ngraph;
 
+// ------------------------------ V1 ------------------------------
+
 TEST(type_prop, proposal_invalid_class_probs_rank)
 {
     op::ProposalAttrs attrs;
     auto class_probs = make_shared<op::Parameter>(element::f32, Shape{1, 2, 3});
-    auto class_logits = make_shared<op::Parameter>(element::f32, Shape{1, 2, 3, 4});
+    auto class_bbox_deltas = make_shared<op::Parameter>(element::f32, Shape{1, 2, 3, 4});
     auto image_shape = make_shared<op::Parameter>(element::f32, Shape{3});
 
     try
     {
-        auto proposal = make_shared<op::Proposal>(class_probs, class_logits, image_shape, attrs);
+        auto proposal =
+            make_shared<op::Proposal>(class_probs, class_bbox_deltas, image_shape, attrs);
```

Comment:
Please, update the test to specify the exact version of the operation Proposal to create.

Reply:
I will do this.

Reply:
Ok, I will do it.

Reply:
Done.